app/controllers/api/users_controller.rb

```diff
@@ -6,6 +6,7 @@ def create
     @new_user.secure_random
     if @new_user.save
       render json: {authorization: @new_user.authorization_token}
+      NewUserMailer.sign_up_email(@new_user).deliver_now
```

Comment:
Do we really want this to block?   Could we break this work out into a deliver_later